torch/csrc/jit/passes/quantization/helper.h

```diff
@@ -89,7 +95,37 @@ findChildModule(const Module& module, const std::vector<std::string>& path);
 // Given an CallMethod node, get the module instance corresponding
 // to the instance Value
 TORCH_API Module getInvokedModule(Module& module, Node* n, Value* self);
-// =========== helper functions for Module  ==========
+
+// ==================== filter functions for matches ==============
+auto aten_add_alpha_is_one =
```

Comment:
How about a general matcher that checks if a particular argument is hardcoded to an integer value?
```c++
    auto check_constant_argument =
[](const Match& match,
       const std::unordered_map<std::string, Value*>& vmap, std::string argument_name, int argument_value) {
      const auto& match_vmap = match.values_map;
      auto alpha = toIValue(match_vmap.at(vmap.at(argument_name)));
      return alpha && alpha->isInt() && alpha->toInt() == argument_value;
    };